plotting/basemap.py

```diff
@@ -16,27 +17,23 @@ def get_resolution(height: float, width: float) -> str:
     area = height * width / 1e6
 
     if area < 1e5:
-        return "f"  # full resolution
+        return "10m"  # full resolution
     elif area < 1e12:
-        return "i"  # intermediate resolution
+        return "50m"  # intermediate resolution
     else:
-        return "c"  # crude resolution
+        return "110m"  # crude resolution
 
 
 def _get_land_geoms(resolution: str, extent: list) -> shpreader.BasicReader:
 
-    if resolution == "f":
-        land_shp = shpreader.BasicReader(
-            current_app.config["SHAPE_FILE_DIR"] + "/cartopy_resources/ne_10m_diff.shp"
-        )
-    elif resolution == "i":
-        land_shp = shpreader.BasicReader(
-            current_app.config["SHAPE_FILE_DIR"] + "/cartopy_resources/ne_50m_diff.shp"
-        )
-    else:
+    shp_file = f"/cartopy_resources/ne_{resolution}_diff.shp"
+
+    try:
         land_shp = shpreader.BasicReader(
-            current_app.config["SHAPE_FILE_DIR"] + "/cartopy_resources/ne_110m_diff.shp"
+            current_app.config["SHAPE_FILE_DIR"] + shp_file
         )
+    except shpreader.shapefile.ShapefileException:
+        land_shp = cfeature.NaturalEarthFeature("physical", "land", resolution)
```

Comment:
nit: could you please add a basic print statement here so we can capture exception case in our rudimentary logs? Something like "using cartopy built-in image as fallback" or whatever.